contents/handbook/people/onboarding.md

```diff
@@ -158,3 +158,14 @@ We use GitHub Projects to track the status of Issues in an easily viewable way.
 You can also assign an Issue to a specific person, and tag it with a relevant label - use these to help people filter more easily.
 
 Each Small Team has its own Project for tracking their Issues - [full list here](https://github.com/orgs/PostHog/projects). Most teams run two week sprints - as part of onboarding, you will be invited to the relevant planning meetings. 
+
+## 30/60/90 day check-ins
+
+As part of the onboarding checklist the ops team will schedule reminders for a new employees manager at the 30, 60 & 90 day mark to simply serve as a reminder that these checkpoints have arrived. There is no formal requirement for a manager to do anything different at these stages but there are a few helpful suggestions below to help the employee experience and to make sure everything is on track with the onboarding and the [first 3 months probationary period](https://posthog.com/handbook/people/compensation#notice-period) will be passed.
+
+- Around the 30 day mark it's good for the manager to provide initial feedback especially if there is constructive feedback that needs to be given to ensure the person passes probation. Separately, it's also a good time to reinforce the positive work that has been done by somebody on the right track.
+- Around the 60 day mark if things are going well, the manager might want to give an indication of this as it can ease any looming fears. A member of the Ops team will reach out at this time to seek clarity with the manager on whether there are any concerns of the new employee not passing their probation.
```

Comment:
```suggestion
- Around the 60 day mark, if things are going well, the manager might want to give an indication of this as it can ease any fears the team member may have. A member of the Ops team will also check in with the manager to see if things are on track.
```